src/analysis/analytic/isolated_zeros.lean

```diff
@@ -137,4 +140,41 @@ begin
   { exact or.inr (hp.locally_ne_zero h) }
 end
 
+lemma frequently_zero_iff_eventually_zero {f : 𝕜 → E} {w : 𝕜} (hf : analytic_at 𝕜 f w) :
+  (∃ᶠ z in 𝓝[≠] w, f z = 0) ↔ (∀ᶠ z in 𝓝 w, f z = 0) :=
+⟨λ h, hf.eventually_eq_zero_or_eventually_ne_zero.cases_on id (false.elim ∘ h),
```

Comment:
should this be `resolve_left` or something

Reply:
Possibly, I just learned about `resolve_left` on zulip. But there would probably still be some wrangling of `nhds_within` versus `nhds`. I don't really mind either way.

Reply:
One implication was `resolve_right` :-)